### DISCUSSION TITLE
what renames my workspaces?

### DISCUSSION THREAD

**Discussion by tve** [Manjaro-Sway/manjaro-sway#352] [Q&A] [answered]
I'm trying to assign names to my workspaces by the names get overwritten by font-icons as soon as a window is created. What does this and how do I disable it?
Eg, from `swaymsg -t get_workspaces` output when I create a workspace it has the name I assigned:
```
Workspace 6:foo (focused)
```
but then I opened firefox and foot and the workspace name changed to:
![image](https://user-images.githubusercontent.com/39480/201490742-e3e8634c-2038-4f5e-9bd8-c9312663d6bf.png)

**Comment by boredland** [comment, marked as answer]
hi! that is [sworkstyle](https://github.com/Lyr-7D1h/swayest_workstyle). if you don't like it, you should be able to [uninstall](https://github.com/manjaro-sway/manjaro-sway/blob/main/SUPPORT.md#how-can-i-disable-the-dynamic-workspace-icons) it.

**Reply by tve** [reply]
oh man, 'been tracking that down for an hour now.. thanks!
